tests/src/cgeo/geocaching/files/ProgressInputStreamTest.java

```diff
@@ -9,7 +9,7 @@
 public class ProgressInputStreamTest extends TestCase {
 
     public static void testRead() throws Exception {
-        ProgressInputStream stream = new ProgressInputStream(IOUtils.toInputStream("test"));
+        ProgressInputStream stream = new ProgressInputStream(IOUtils.toInputStream("test", (String) null));
```

Comment:
Thanks for developing @Bananeweizen, that was indeed my point entirely. This is fine here whatever the default is (I think even Asian language contain the basic latin plane), but I would not want to see a change in the string made later with consequences differing depending on the default locale.